### DISCUSSION TITLE
why don't my commits appear in the commit graph? att: they were uploaded and registered normally

### DISCUSSION THREAD

**Discussion by marcosvtenorio** [community/community#33414] [Other Feature Feedback, Questions, & Ideas] [answered]
and i check all the requirements:

The email address used for commits is associated with your GitHub.com account... CHECK
The commits were created in a standalone repository, not a fork... CHECK
You are a contributor to the repository or a member of the organization that owns the repository... CHECK
You bookmarked the repository... CHECK

His on branch main
Is only me in this repository...

HELP MEEEE

**Comment by radar-blip** [comment, marked as answer]
This might help if your repository is private. Check in settings for this:
![profiles](https://user-images.githubusercontent.com/39196767/190923060-3bf650c6-465b-4392-9fd6-6163f8d6332e.png)
